contactaccessrights/tests/phpunit/CRM/Contactaccessrights/Utils/ACLTest.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+use Civi\Test\HeadlessInterface;
+use Civi\Test\TransactionalInterface;
+use CRM_HRCore_Test_Fabricator_Contact as ContactFabricator;
+use CRM_Hrjobcontract_Test_Fabricator_HRJobContract as JobContractFabricator;
+use CRM_Contactaccessrights_Test_Fabricator_Rights as RightsFabricator;
+
+/**
+ * Class CRM_Contactaccessrights_BAO_RightsTest
+ * Tests Rights BAO class for contact's access rights to civicrm.
+ * 
+ * @group headless
+ * @author Camilo Rodriguez - camilo@compucorp.co.uk

```

Comment:
No need for the @author tag here as well (If these are being added by your editor, you can configure it to not add them automatically)